src/olympia/abuse/cinder.py

```diff
@@ -56,27 +56,35 @@ def get_extended_attributes(self):
         # are typically returned there instead of in get_attributes().
         return {}
 
-    def build_report_payload(self, *, report_text, category, reporter):
+    def build_report_payload(self, *, report, reporter):
         context = self.get_context()
-        if reporter:
-            reporter_entity_data = reporter.get_entity_data()
-            if reporter_entity_data not in context['entities']:
-                context['entities'] += [reporter.get_entity_data()]
+        if report:
```

Comment:
I'm not sure about the use-case of `report` not being provided but 🤷‍♂️

Reply:
Just didn't want to fail completely initially when I was making the changes. I kept it because I figured we might want to be able to build a payload without a `report` for reviewer actions later